cibyl/cli/query.py

```diff
@@ -72,34 +72,34 @@ def get_query_type_core(self, command: Optional[str] = None,
         result = QueryType.NONE
 
         if 'tenants' in kwargs:
-            result = QueryType.TENANTS
+            result |= QueryType.TENANTS
 
         if 'projects' in kwargs:
-            result = QueryType.PROJECTS
+            result |= QueryType.PROJECTS
 
         if 'pipelines' in kwargs:
-            result = QueryType.PIPELINES
+            result |= QueryType.PIPELINES
 
         job_args = subset(kwargs, ["jobs", "job_url"])
         if job_args:
-            result = QueryType.JOBS
+            result |= QueryType.JOBS
 
         if 'variants' in kwargs:
-            result = QueryType.VARIANTS
+            result |= QueryType.VARIANTS
 
         build_args = subset(kwargs, ["builds", "last_build", "build_status"])
         if build_args:
-            result = QueryType.BUILDS
+            result |= QueryType.BUILDS
 
         test_args = subset(kwargs, ["tests", "test_result", "test_duration"])
         if test_args:
-            result = QueryType.TESTS
+            result |= QueryType.TESTS
 
         if command == 'features':
             if job_args:
-                result = QueryType.FEATURES_JOBS
+                result |= QueryType.FEATURES_JOBS
             else:
-                result = QueryType.FEATURES
+                result |= QueryType.FEATURES
 
         return result
 
```

Comment:
The `get_type_query` method below should also be updated to use an OR operator instead of `max`. Other than that, I think this is a great update for the QueryType, makes it much more useful. Good idea!

Reply:
Then, instead of picking between cibyl's or the plugin's query, you would like to add the two of them together, right?

Reply:
At this point the 'max' indicates that the most complex query will get chosen.

Reply:
> Then, instead of picking between cibyl's or the plugin's query, you would like to add the two of them together, right?

Exactly, that way a command like `cibyl query --pipelines --ip-version` will result in a QueryType `JOBS|PIPELINES

Reply:
All right, done.